{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "6f401209",
   "metadata": {
    "pycharm": {
     "name": "#%%\n"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85041394976'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85041394976?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85041394976&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85041394976&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85041394976', 'dc:identifier': 'SCOPUS_ID:85041394976', 'eid': '2-s2.0-85041394976', 'dc:title': 'SCANPY: Large-scale single-cell gene expression data analysis', 'dc:creator': 'Wolf F.A.', 'prism:publicationName': 'Genome Biology', 'prism:issn': '14747596', 'prism:eIssn': '1474760X', 'prism:volume': '19', 'prism:issueIdentifier': '1', 'prism:pageRange': None, 'prism:coverDate': '2018-02-06', 'prism:coverDisplayDate': '6 February 2018', 'prism:doi': '10.1186/s13059-017-1382-0', 'citedby-count': '975', 'affiliation': [{'@_fa': 'true', 'affilname': 'Helmholtz Center Munich German Research Center for Environmental Health', 'affiliation-city': 'Neuherberg', 'affiliation-country': 'Germany'}], 'pubmed-id': '29409532', 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': '15', 'source-id': '40944', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/50149119645'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/50149119645?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=50149119645&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=50149119645&origin=inward'}, {'@_fa': 'true', '@ref': 'full-text', '@href': 'https://api.elsevier.com/content/article/eid/1-s2.0-S0022519308003020'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/50149119645', 'dc:identifier': 'SCOPUS_ID:50149119645', 'eid': '2-s2.0-50149119645', 'dc:title': 'Enzymes/non-enzymes classification model complexity based on composition, sequence, 3D and topological indices', 'dc:creator': 'Munteanu C.R.', 'prism:publicationName': 'Journal of Theoretical Biology', 'prism:issn': '00225193', 'prism:eIssn': '10958541', 'prism:volume': '254', 'prism:issueIdentifier': '2', 'prism:pageRange': '476-482', 'prism:coverDate': '2008-09-21', 'prism:coverDisplayDate': '21 September 2008', 'prism:doi': '10.1016/j.jtbi.2008.06.003', 'pii': 'S0022519308003020', 'citedby-count': '68', 'affiliation': [{'@_fa': 'true', 'affilname': 'Universidade do Porto', 'affiliation-city': 'Porto', 'affiliation-country': 'Portugal'}], 'pubmed-id': '18606172', 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '29663', 'openaccess': '0', 'openaccessFlag': False}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85012012609'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85012012609?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85012012609&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85012012609&origin=inward'}, {'@_fa': 'true', '@ref': 'full-text', '@href': 'https://api.elsevier.com/content/article/eid/1-s2.0-S016816991630494X'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85012012609', 'dc:identifier': 'SCOPUS_ID:85012012609', 'eid': '2-s2.0-85012012609', 'dc:title': 'Weed detecting robot in sugarcane fields using fuzzy real time classifier', 'dc:creator': 'Sujaritha M.', 'prism:publicationName': 'Computers and Electronics in Agriculture', 'prism:issn': '01681699', 'prism:volume': '134', 'prism:pageRange': '160-171', 'prism:coverDate': '2017-03-01', 'prism:coverDisplayDate': '1 March 2017', 'prism:doi': '10.1016/j.compag.2017.01.008', 'pii': 'S016816991630494X', 'citedby-count': '51', 'affiliation': [{'@_fa': 'true', 'affilname': 'Sri Krishna College of Engineering and Technology, Coimbatore', 'affiliation-city': 'Coimbatore', 'affiliation-country': 'India'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '30441', 'openaccess': '0', 'openaccessFlag': False}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/79952327303'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/79952327303?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=79952327303&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=79952327303&origin=inward'}, {'@_fa': 'true', '@ref': 'full-text', '@href': 'https://api.elsevier.com/content/article/eid/1-s2.0-S0022519311000221'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/79952327303', 'dc:identifier': 'SCOPUS_ID:79952327303', 'eid': '2-s2.0-79952327303', 'dc:title': 'NL MIND-BEST: A web server for ligands and proteins discovery-Theoretic-experimental study of proteins of Giardia lamblia and new compounds active against Plasmodium falciparum', 'dc:creator': 'González-Díaz H.', 'prism:publicationName': 'Journal of Theoretical Biology', 'prism:issn': '00225193', 'prism:eIssn': '10958541', 'prism:volume': '276', 'prism:issueIdentifier': '1', 'prism:pageRange': '229-249', 'prism:coverDate': '2011-05-07', 'prism:coverDisplayDate': '7 May 2011', 'prism:doi': '10.1016/j.jtbi.2011.01.010', 'pii': 'S0022519311000221', 'citedby-count': '49', 'affiliation': [{'@_fa': 'true', 'affilname': 'Universidad de Santiago de Compostela', 'affiliation-city': 'Santiago de Compostela', 'affiliation-country': 'Spain'}], 'pubmed-id': '21277861', 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '29663', 'openaccess': '0', 'openaccessFlag': False, 'freetoread': {'value': [{'$': 'all'}, {'$': 'repository'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85081395376'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85081395376?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85081395376&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85081395376&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85081395376', 'dc:identifier': 'SCOPUS_ID:85081395376', 'eid': '2-s2.0-85081395376', 'dc:title': 'BrainSpace: a toolbox for the analysis of macroscale gradients in neuroimaging and connectomics datasets', 'dc:creator': 'Vos de Wael R.', 'prism:publicationName': 'Communications Biology', 'prism:eIssn': '23993642', 'prism:volume': '3', 'prism:issueIdentifier': '1', 'prism:pageRange': None, 'prism:coverDate': '2020-12-01', 'prism:coverDisplayDate': '1 December 2020', 'prism:doi': '10.1038/s42003-020-0794-7', 'citedby-count': '47', 'affiliation': [{'@_fa': 'true', 'affilname': 'McConnell Brain Imaging Centre', 'affiliation-city': 'Montreal', 'affiliation-country': 'Canada'}], 'pubmed-id': '32139786', 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': '103', 'source-id': '21100924827', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85044577091'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85044577091?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85044577091&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85044577091&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85044577091', 'dc:identifier': 'SCOPUS_ID:85044577091', 'eid': '2-s2.0-85044577091', 'dc:title': 'Pluvial Flood Risk Assessment Tool (PFRA) for rainwater management and adaptation to climate change in newly urbanised areas', 'dc:creator': 'Szewrański S.', 'prism:publicationName': 'Water (Switzerland)', 'prism:eIssn': '20734441', 'prism:volume': '10', 'prism:issueIdentifier': '4', 'prism:pageRange': None, 'prism:coverDate': '2018-03-26', 'prism:coverDisplayDate': '26 March 2018', 'prism:doi': '10.3390/w10040386', 'citedby-count': '41', 'affiliation': [{'@_fa': 'true', 'affilname': 'Wrocław University of Environmental and Life Sciences', 'affiliation-city': 'Wroclaw', 'affiliation-country': 'Poland'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': '386', 'source-id': '21100255400', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/84978886780'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/84978886780?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=84978886780&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=84978886780&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/84978886780', 'dc:identifier': 'SCOPUS_ID:84978886780', 'eid': '2-s2.0-84978886780', 'dc:title': 'An Introduction to Programming for Bioscientists: A Python-Based Primer', 'dc:creator': 'Ekmekci B.', 'prism:publicationName': 'PLoS Computational Biology', 'prism:issn': '1553734X', 'prism:eIssn': '15537358', 'prism:volume': '12', 'prism:issueIdentifier': '6', 'prism:pageRange': None, 'prism:coverDate': '2016-06-01', 'prism:coverDisplayDate': 'June 2016', 'prism:doi': '10.1371/journal.pcbi.1004867', 'citedby-count': '35', 'affiliation': [{'@_fa': 'true', 'affilname': 'University of Virginia', 'affiliation-city': 'Charlottesville', 'affiliation-country': 'United States'}], 'pubmed-id': '27271528', 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': 'e1004867', 'source-id': '4000151810', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85038902686'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85038902686?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85038902686&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85038902686&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85038902686', 'dc:identifier': 'SCOPUS_ID:85038902686', 'eid': '2-s2.0-85038902686', 'dc:title': 'Leaf-GP: An open and automated software application for measuring growth phenotypes for arabidopsis and wheat', 'dc:creator': 'Zhou J.', 'prism:publicationName': 'Plant Methods', 'prism:eIssn': '17464811', 'prism:volume': '13', 'prism:issueIdentifier': '1', 'prism:pageRange': None, 'prism:coverDate': '2017-12-22', 'prism:coverDisplayDate': '22 December 2017', 'prism:doi': '10.1186/s13007-017-0266-3', 'citedby-count': '25', 'affiliation': [{'@_fa': 'true', 'affilname': 'Earlham Institute', 'affiliation-city': 'Norwich', 'affiliation-country': 'United Kingdom'}, {'@_fa': 'true', 'affilname': 'John Innes Centre', 'affiliation-city': 'Norwich', 'affiliation-country': 'United Kingdom'}, {'@_fa': 'true', 'affilname': 'University of East Anglia', 'affiliation-city': 'Norwich', 'affiliation-country': 'United Kingdom'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': '117', 'source-id': '6400153153', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85055108721'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85055108721?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85055108721&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85055108721&origin=inward'}, {'@_fa': 'true', '@ref': 'full-text', '@href': 'https://api.elsevier.com/content/article/eid/1-s2.0-S0006320718303379'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85055108721', 'dc:identifier': 'SCOPUS_ID:85055108721', 'eid': '2-s2.0-85055108721', 'dc:title': 'Text and data mining of social media to map wildlife recreation activity', 'dc:creator': 'Monkman G.', 'prism:publicationName': 'Biological Conservation', 'prism:issn': '00063207', 'prism:volume': '228', 'prism:pageRange': '89-99', 'prism:coverDate': '2018-12-01', 'prism:coverDisplayDate': 'December 2018', 'prism:doi': '10.1016/j.biocon.2018.10.010', 'pii': 'S0006320718303379', 'citedby-count': '24', 'affiliation': [{'@_fa': 'true', 'affilname': 'Bangor University', 'affiliation-city': 'Bangor', 'affiliation-country': 'United Kingdom'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '17244', 'openaccess': '0', 'openaccessFlag': False}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85089320896'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85089320896?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85089320896&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85089320896&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85089320896', 'dc:identifier': 'SCOPUS_ID:85089320896', 'eid': '2-s2.0-85089320896', 'dc:title': 'Predicting antigen specificity of single T cells based\\xa0on TCR CDR3 regions', 'dc:creator': 'Fischer D.S.', 'prism:publicationName': 'Molecular Systems Biology', 'prism:eIssn': '17444292', 'prism:volume': '16', 'prism:issueIdentifier': '8', 'prism:pageRange': None, 'prism:coverDate': '2020-08-01', 'prism:coverDisplayDate': '1 August 2020', 'prism:doi': '10.15252/msb.20199416', 'citedby-count': '20', 'affiliation': [{'@_fa': 'true', 'affilname': 'Wissenschaftszentrum Weihenstephan für Ernährung, Landnutzung und Umwelt', 'affiliation-city': 'Freising', 'affiliation-country': 'Germany'}, {'@_fa': 'true', 'affilname': 'Helmholtz Center Munich German Research Center for Environmental Health', 'affiliation-city': 'Neuherberg', 'affiliation-country': 'Germany'}], 'pubmed-id': '32779888', 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': 'e9416', 'source-id': '4700152228', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85059123356'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85059123356?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85059123356&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85059123356&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85059123356', 'dc:identifier': 'SCOPUS_ID:85059123356', 'eid': '2-s2.0-85059123356', 'dc:title': 'A novel ArcGIS toolbox for estimating crop water demands by integrating the dual crop coefficient approach with multi-satellite imagery', 'dc:creator': 'Ramírez-Cuesta J.M.', 'prism:publicationName': 'Water (Switzerland)', 'prism:eIssn': '20734441', 'prism:volume': '11', 'prism:issueIdentifier': '1', 'prism:pageRange': None, 'prism:coverDate': '2018-12-25', 'prism:coverDisplayDate': '25 December 2018', 'prism:doi': '10.3390/w11010038', 'citedby-count': '20', 'affiliation': [{'@_fa': 'true', 'affilname': 'CEBAS- CSIC, Centro de Edafología y Biología Aplicada del Segura', 'affiliation-city': 'Murcia', 'affiliation-country': 'Spain'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': '38', 'source-id': '21100255400', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85055913635'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85055913635?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85055913635&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85055913635&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85055913635', 'dc:identifier': 'SCOPUS_ID:85055913635', 'eid': '2-s2.0-85055913635', 'dc:title': 'A k-mer-based method for the identification of phenotype-associated genomic biomarkers and predicting phenotypes of sequenced bacteria', 'dc:creator': 'Aun E.', 'prism:publicationName': 'PLoS Computational Biology', 'prism:issn': '1553734X', 'prism:eIssn': '15537358', 'prism:volume': '14', 'prism:issueIdentifier': '10', 'prism:pageRange': None, 'prism:coverDate': '2018-10-01', 'prism:coverDisplayDate': 'October 2018', 'prism:doi': '10.1371/journal.pcbi.1006434', 'citedby-count': '18', 'affiliation': [{'@_fa': 'true', 'affilname': 'Tartu Ülikool', 'affiliation-city': 'Tartu', 'affiliation-country': 'Estonia'}], 'pubmed-id': '30346947', 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': 'e1006434', 'source-id': '4000151810', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85097789181'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85097789181?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85097789181&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85097789181&origin=inward'}, {'@_fa': 'true', '@ref': 'full-text', '@href': 'https://api.elsevier.com/content/article/eid/1-s2.0-S003442572030609X'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85097789181', 'dc:identifier': 'SCOPUS_ID:85097789181', 'eid': '2-s2.0-85097789181', 'dc:title': 'OC-SMART: A machine learning based data analysis platform for satellite ocean color sensors', 'dc:creator': 'Fan Y.', 'prism:publicationName': 'Remote Sensing of Environment', 'prism:issn': '00344257', 'prism:volume': '253', 'prism:pageRange': None, 'prism:coverDate': '2021-02-01', 'prism:coverDisplayDate': 'February 2021', 'prism:doi': '10.1016/j.rse.2020.112236', 'pii': 'S003442572030609X', 'citedby-count': '17', 'affiliation': [{'@_fa': 'true', 'affilname': 'Stevens Institute of Technology', 'affiliation-city': 'Hoboken', 'affiliation-country': 'United States'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': '112236', 'source-id': '12503', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfree2read'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Bronze'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85050818943'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85050818943?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85050818943&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85050818943&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85050818943', 'dc:identifier': 'SCOPUS_ID:85050818943', 'eid': '2-s2.0-85050818943', 'dc:title': 'A global analysis of bats using automated comparative phylogeography uncovers a surprising impact of Pleistocene glaciation', 'dc:creator': 'Carstens B.C.', 'prism:publicationName': 'Journal of Biogeography', 'prism:issn': '03050270', 'prism:eIssn': '13652699', 'prism:volume': '45', 'prism:issueIdentifier': '8', 'prism:pageRange': '1795-1805', 'prism:coverDate': '2018-08-01', 'prism:coverDisplayDate': 'August 2018', 'prism:doi': '10.1111/jbi.13382', 'citedby-count': '16', 'affiliation': [{'@_fa': 'true', 'affilname': 'The Ohio State University', 'affiliation-city': 'Columbus', 'affiliation-country': 'United States'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '29269', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfree2read'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Bronze'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85090972696'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85090972696?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85090972696&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85090972696&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85090972696', 'dc:identifier': 'SCOPUS_ID:85090972696', 'eid': '2-s2.0-85090972696', 'dc:title': 'Sorghum Panicle Detection and Counting Using Unmanned Aerial System Images and Deep Learning', 'dc:creator': 'Lin Z.', 'prism:publicationName': 'Frontiers in Plant Science', 'prism:eIssn': '1664462X', 'prism:volume': '11', 'prism:pageRange': None, 'prism:coverDate': '2020-09-02', 'prism:coverDisplayDate': '2 September 2020', 'prism:doi': '10.3389/fpls.2020.534853', 'citedby-count': '15', 'affiliation': [{'@_fa': 'true', 'affilname': 'Texas Tech University', 'affiliation-city': 'Lubbock', 'affiliation-country': 'United States'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': '534853', 'source-id': '21100313905', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85068710230'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85068710230?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85068710230&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85068710230&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85068710230', 'dc:identifier': 'SCOPUS_ID:85068710230', 'eid': '2-s2.0-85068710230', 'dc:title': 'Quantification of uncertainty in 3-D seismic interpretation: Implications for deterministic and stochastic geomodeling and machine learning', 'dc:creator': 'Schaaf A.', 'prism:publicationName': 'Solid Earth', 'prism:issn': '18699510', 'prism:eIssn': '18699529', 'prism:volume': '10', 'prism:issueIdentifier': '4', 'prism:pageRange': '1049-1061', 'prism:coverDate': '2019-07-05', 'prism:coverDisplayDate': '5 July 2019', 'prism:doi': '10.5194/se-10-1049-2019', 'citedby-count': '15', 'affiliation': [{'@_fa': 'true', 'affilname': 'University of Aberdeen', 'affiliation-city': 'Aberdeen', 'affiliation-country': 'United Kingdom'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '19900191969', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85054340081'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85054340081?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85054340081&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85054340081&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85054340081', 'dc:identifier': 'SCOPUS_ID:85054340081', 'eid': '2-s2.0-85054340081', 'dc:title': 'Finding turning-points in ultra-high-resolution animal movement data', 'dc:creator': 'Potts J.R.', 'prism:publicationName': 'Methods in Ecology and Evolution', 'prism:eIssn': '2041210X', 'prism:volume': '9', 'prism:issueIdentifier': '10', 'prism:pageRange': '2091-2101', 'prism:coverDate': '2018-10-01', 'prism:coverDisplayDate': 'October 2018', 'prism:doi': '10.1111/2041-210X.13056', 'citedby-count': '15', 'affiliation': [{'@_fa': 'true', 'affilname': 'The University of Sheffield', 'affiliation-city': 'Sheffield', 'affiliation-country': 'United Kingdom'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '21100217611', 'openaccess': '0', 'openaccessFlag': False, 'freetoread': {'value': [{'$': 'all'}, {'$': 'repository'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/84907588544'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/84907588544?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=84907588544&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=84907588544&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/84907588544', 'dc:identifier': 'SCOPUS_ID:84907588544', 'eid': '2-s2.0-84907588544', 'dc:title': 'IDEPI: Rapid Prediction of HIV-1 Antibody Epitopes and Other Phenotypic Features from Sequence Data Using a Flexible Machine Learning Platform', 'dc:creator': 'Hepler N.L.', 'prism:publicationName': 'PLoS Computational Biology', 'prism:issn': '1553734X', 'prism:eIssn': '15537358', 'prism:volume': '10', 'prism:issueIdentifier': '9', 'prism:pageRange': None, 'prism:coverDate': '2014-09-01', 'prism:coverDisplayDate': '1 September 2014', 'prism:doi': '10.1371/journal.pcbi.1003842', 'citedby-count': '15', 'affiliation': [{'@_fa': 'true', 'affilname': 'University of California, San Diego', 'affiliation-city': 'San Diego', 'affiliation-country': 'United States'}], 'pubmed-id': '25254639', 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '4000151810', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85077941594'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85077941594?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85077941594&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85077941594&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85077941594', 'dc:identifier': 'SCOPUS_ID:85077941594', 'eid': '2-s2.0-85077941594', 'dc:title': 'BrainIAK tutorials: User-friendly learning materials for advanced fMRI analysis', 'dc:creator': 'Kumar M.', 'prism:publicationName': 'PLoS computational biology', 'prism:eIssn': '15537358', 'prism:volume': '16', 'prism:issueIdentifier': '1', 'prism:pageRange': 'e1007549', 'prism:coverDate': '2020-01-01', 'prism:coverDisplayDate': '1 January 2020', 'prism:doi': '10.1371/journal.pcbi.1007549', 'citedby-count': '13', 'affiliation': [{'@_fa': 'true', 'affilname': 'Princeton University', 'affiliation-city': 'Princeton', 'affiliation-country': 'United States'}], 'pubmed-id': '31940340', 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '4000151810', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85077046122'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85077046122?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85077046122&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85077046122&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85077046122', 'dc:identifier': 'SCOPUS_ID:85077046122', 'eid': '2-s2.0-85077046122', 'dc:title': 'Size classification of tomato fruit using thresholding, machine learning and deep learning techniques', 'dc:creator': 'de Luna R.', 'prism:publicationName': 'Agrivita', 'prism:issn': '01260537', 'prism:eIssn': '24778516', 'prism:volume': '41', 'prism:issueIdentifier': '3', 'prism:pageRange': '586-596', 'prism:coverDate': '2019-01-01', 'prism:coverDisplayDate': '2019', 'prism:doi': '10.17503/agrivita.v41i3.2435', 'citedby-count': '11', 'affiliation': [{'@_fa': 'true', 'affilname': 'De La Salle University', 'affiliation-city': 'Manila', 'affiliation-country': 'Philippines'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '21100242817', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85087145975'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85087145975?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85087145975&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85087145975&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85087145975', 'dc:identifier': 'SCOPUS_ID:85087145975', 'eid': '2-s2.0-85087145975', 'dc:title': 'Digitally deconstructing leaves in 3D using X-ray microcomputed tomography and machine learning', 'dc:creator': 'Théroux-Rancourt G.', 'prism:publicationName': 'Applications in Plant Sciences', 'prism:issn': '21680450', 'prism:eIssn': '15372197', 'prism:volume': '8', 'prism:issueIdentifier': '7', 'prism:pageRange': None, 'prism:coverDate': '2020-07-01', 'prism:coverDisplayDate': '1 July 2020', 'prism:doi': '10.1002/aps3.11380', 'citedby-count': '10', 'affiliation': [{'@_fa': 'true', 'affilname': 'Universitat fur Bodenkultur Wien', 'affiliation-city': 'Vienna', 'affiliation-country': 'Austria'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': 'e11380', 'source-id': '21100403506', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85015994482'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85015994482?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85015994482&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85015994482&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85015994482', 'dc:identifier': 'SCOPUS_ID:85015994482', 'eid': '2-s2.0-85015994482', 'dc:title': 'ASAP: A machine learning framework for local protein properties', 'dc:creator': 'Brandes N.', 'prism:publicationName': 'Database', 'prism:issn': '17580463', 'prism:volume': '2016', 'prism:pageRange': None, 'prism:coverDate': '2016-01-01', 'prism:coverDisplayDate': '1 January 2016', 'prism:doi': '10.1093/database/baw133', 'citedby-count': '10', 'affiliation': [{'@_fa': 'true', 'affilname': 'Hebrew University of Jerusalem', 'affiliation-city': 'Jerusalem', 'affiliation-country': 'Israel'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': 'baw133', 'source-id': '19700188321', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/84935443132'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/84935443132?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=84935443132&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=84935443132&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/84935443132', 'dc:identifier': 'SCOPUS_ID:84935443132', 'eid': '2-s2.0-84935443132', 'dc:title': 'Multi-level machine learning prediction of protein-protein interactions in Saccharomyces cerevisiae', 'dc:creator': 'Zubek J.', 'prism:publicationName': 'PeerJ', 'prism:eIssn': '21678359', 'prism:volume': '2015', 'prism:issueIdentifier': '7', 'prism:pageRange': None, 'prism:coverDate': '2015-01-01', 'prism:coverDisplayDate': '2015', 'prism:doi': '10.7717/peerj.1041', 'citedby-count': '9', 'affiliation': [{'@_fa': 'true', 'affilname': 'Centrum Nowych Technologii Uniwersytetu Warszawskiego', 'affiliation-city': 'Warsaw', 'affiliation-country': 'Poland'}, {'@_fa': 'true', 'affilname': 'Institute of Computer Science of the Polish Academy of Sciences', 'affiliation-city': 'Warsaw', 'affiliation-country': 'Poland'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': '1041', 'source-id': '21100239256', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}, {'$': 'repository'}, {'$': 'repositoryvor'}, {'$': 'repositoryam'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}, {'$': 'Green'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85108944689'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85108944689?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85108944689&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85108944689&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85108944689', 'dc:identifier': 'SCOPUS_ID:85108944689', 'eid': '2-s2.0-85108944689', 'dc:title': 'Analysis of the forecast price as a factor of sustainable development of agriculture', 'dc:creator': 'Tatarintsev M.', 'prism:publicationName': 'Agronomy', 'prism:eIssn': '20734395', 'prism:volume': '11', 'prism:issueIdentifier': '6', 'prism:pageRange': None, 'prism:coverDate': '2021-06-01', 'prism:coverDisplayDate': 'June 2021', 'prism:doi': '10.3390/agronomy11061235', 'citedby-count': '8', 'affiliation': [{'@_fa': 'true', 'affilname': 'Financial University under the Government of the Russian Federation', 'affiliation-city': 'Moscow', 'affiliation-country': 'Russian Federation'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'article-number': '1235', 'source-id': '21100447811', 'openaccess': '1', 'openaccessFlag': True, 'freetoread': {'value': [{'$': 'all'}, {'$': 'publisherfullgold'}]}, 'freetoreadLabel': {'value': [{'$': 'All Open Access'}, {'$': 'Gold'}]}}, {'@_fa': 'true', 'link': [{'@_fa': 'true', '@ref': 'self', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85088993216'}, {'@_fa': 'true', '@ref': 'author-affiliation', '@href': 'https://api.elsevier.com/content/abstract/scopus_id/85088993216?field=author,affiliation'}, {'@_fa': 'true', '@ref': 'scopus', '@href': 'https://www.scopus.com/inward/record.uri?partnerID=HzOxMe3b&scp=85088993216&origin=inward'}, {'@_fa': 'true', '@ref': 'scopus-citedby', '@href': 'https://www.scopus.com/inward/citedby.uri?partnerID=HzOxMe3b&scp=85088993216&origin=inward'}], 'prism:url': 'https://api.elsevier.com/content/abstract/scopus_id/85088993216', 'dc:identifier': 'SCOPUS_ID:85088993216', 'eid': '2-s2.0-85088993216', 'dc:title': 'Prediction of heart disease by classifying with feature selection and machine learning methods', 'dc:creator': 'Gazeloğlu C.', 'prism:publicationName': 'Progress in Nutrition', 'prism:issn': '11298723', 'prism:volume': '22', 'prism:issueIdentifier': '2', 'prism:pageRange': '660-670', 'prism:coverDate': '2020-01-01', 'prism:coverDisplayDate': '2020', 'prism:doi': '10.23751/pn.v22i2.9830', 'citedby-count': '8', 'affiliation': [{'@_fa': 'true', 'affilname': 'Medical School of Suleyman Demirel Universtiy', 'affiliation-city': 'Isparta', 'affiliation-country': 'Turkey'}], 'prism:aggregationType': 'Journal', 'subtype': 'ar', 'subtypeDescription': 'Article', 'source-id': '27995', 'openaccess': '0', 'openaccessFlag': False}]\n"
     ]
    }
   ],
   "source": [
    "base_url = 'http://api.elsevier.com/content/search/scopus?'\n",
    "# search in title, abstract, and key\n",
    "scope    = 'TITLE-ABS-KEY'\n",
    "# formulating the query structure\n",
    "terms1   = '({python})'\n",
    "terms2   = '({machine learning} OR {big data} OR {artificial intelligence})'\n",
    "terms    = '({} AND {})'.format(terms1, terms2)\n",
    "# insert your personal key (it is free and available on https://dev.elsevier.com/)\n",
    "apiKey   = '&apiKey=5c6dcc9357dd0800c60c4ef571484947'\n",
    "date     = '&date=1990-2022'\n",
    "# it is the maximum number of results per query for a free account\n",
    "count    = '&count=25'\n",
    "sort     = '&sort=citedby-count'\n",
    "view     = '&view=standard'\n",
    "\n",
    "import requests\n",
    "# this function sends a request and returns the total articles, the #starting position of the first article, and the metadata of each #article.\n",
    "import json\n",
    "def search_scopus(url):\n",
    "\n",
    "    res = requests.get(url)\n",
    "    if res.status_code ==200:\n",
    "        content  = json.loads(res.content)['search-results']\n",
    "        # print(content)\n",
    "        total    = content['opensearch:totalResults']\n",
    "        start    = content['opensearch:startIndex']\n",
    "        metadata = content['entry']\n",
    "        return int(total), int(start), metadata\n",
    "\n",
    "    else:\n",
    "        error = json.loads(res.content)['service-error']['status']\n",
    "        print(res.status_code, error['statusText'])\n",
    "# list of all subjects in Scopus database\n",
    "subjects = ['AGRI']\n",
    "for sub in subjects:\n",
    "  while True:\n",
    "\n",
    "    # starting index of the results for display\n",
    "    # starting index refers to number position of not pages\n",
    "    start_index  = 0\n",
    "\n",
    "    start    = '&start={}'.format(start_index)\n",
    "    subj     = '&subj={}'.format(sub)\n",
    "    query    = 'query=' + scope + terms + date + start + count +  sort + subj + apiKey + view\n",
    "    url  = base_url + query\n",
    "    # total results per subject, starting index of first result in\n",
    "    #each query and data\n",
    "    total, start_index, metadata = search_scopus(url)\n",
    "    # save metadata now in SQL (not shown here)\n",
    "    # check how many results need to be retrieved\n",
    "    remain = total - start_index - len(metadata)\n",
    "    print(metadata)\n",
    "    break # breaking from while loop"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "cdf00533",
   "metadata": {
    "pycharm": {
     "name": "#%%\n"
    }
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'scope' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001B[0;31m---------------------------------------------------------------------------\u001B[0m",
      "\u001B[0;31mNameError\u001B[0m                                 Traceback (most recent call last)",
      "Input \u001B[0;32mIn [1]\u001B[0m, in \u001B[0;36m<cell line: 19>\u001B[0;34m()\u001B[0m\n\u001B[1;32m     26\u001B[0m start    \u001B[38;5;241m=\u001B[39m \u001B[38;5;124m'\u001B[39m\u001B[38;5;124m&start=\u001B[39m\u001B[38;5;132;01m{}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;241m.\u001B[39mformat(start_index) \n\u001B[1;32m     27\u001B[0m subj     \u001B[38;5;241m=\u001B[39m \u001B[38;5;124m'\u001B[39m\u001B[38;5;124m&subj=\u001B[39m\u001B[38;5;132;01m{}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;241m.\u001B[39mformat(sub)\n\u001B[0;32m---> 28\u001B[0m query    \u001B[38;5;241m=\u001B[39m \u001B[38;5;124m'\u001B[39m\u001B[38;5;124mquery=\u001B[39m\u001B[38;5;124m'\u001B[39m \u001B[38;5;241m+\u001B[39m \u001B[43mscope\u001B[49m \u001B[38;5;241m+\u001B[39m terms \u001B[38;5;241m+\u001B[39m date \u001B[38;5;241m+\u001B[39m start \u001B[38;5;241m+\u001B[39m count \u001B[38;5;241m+\u001B[39m  sort \u001B[38;5;241m+\u001B[39m subj \u001B[38;5;241m+\u001B[39m apiKey \u001B[38;5;241m+\u001B[39m view\n\u001B[1;32m     29\u001B[0m url  \u001B[38;5;241m=\u001B[39m base_url \u001B[38;5;241m+\u001B[39m query\n\u001B[1;32m     30\u001B[0m \u001B[38;5;66;03m# total results per subject, starting index of first result in \u001B[39;00m\n\u001B[1;32m     31\u001B[0m \u001B[38;5;66;03m#each query and data\u001B[39;00m\n",
      "\u001B[0;31mNameError\u001B[0m: name 'scope' is not defined"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "dataframe = pd.DataFrame()\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "30992219",
   "metadata": {
    "pycharm": {
     "name": "#%%\n"
    }
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}